### DISCUSSION TITLE
why there is delay in filling requests this motnh

### DISCUSSION THREAD

**Discussion by rchiileea** [elax46/custom-brand-icons#820] [Show and tell]
Hi all there is a delay in filling requests this month due to me traveling with a project I helped my newphew with, it is a custom themed pc themed on the hyundai ioniq 5 N (monster of a car), things should return to normal at the end of the month as I have been busy trying to get it more votes here  [https://www.facebook.com/events/1243999376638802?post_id=1244040116634728&view=permalink](https://www.facebook.com/events/1243999376638802/?post_id=1244040116634728&view=permalink) if you fancy voting for 2guys1case to help me free up time :) (shameful i know). 

![IMG_5232](https://github.com/user-attachments/assets/f43b1fe6-f308-4227-aa91-446314d433b7)
![IMG_5221](https://github.com/user-attachments/assets/172cce25-330f-48b7-b95c-adfbea9184a3)
![IMG_5219](https://github.com/user-attachments/assets/024b65f3-2b54-49d8-90b9-9fc66a06a537)
![IMG_5225](https://github.com/user-attachments/assets/3a02e5ab-6d49-406a-8000-923c574c84d3)
![IMG_5284](https://github.com/user-attachments/assets/c83d2599-de06-43c7-b83b-a725e1e7556f)
![IMG_5304](https://github.com/user-attachments/assets/d0ad17b0-40fa-4dbd-b2da-9afbd6483f26)
![IMG_5317](https://github.com/user-attachments/assets/cae4cada-13df-4e6e-add8-d2a89cf2f130)
![IMG_5316](https://github.com/user-attachments/assets/9d24ceba-8d93-4e7b-85e4-4311eec700a8)
![IMG_5345](https://github.com/user-attachments/assets/8712dbb1-4458-4536-989a-f36f9167d146)

**Comment by rchiileea** [comment]
@elax46 can you make this a sticky

**Reply by elax46** [reply]
wow crazy work! pin the discussion

**Comment by rchiileea** [comment]
Ok Guys, home remodel should be finished middle of next month, expect most requests to get back to normal.

**Comment by rchiileea** [comment]
its been hectic, cant wait for it to go back to normal, or should I say it will ever now the wife has decided that as the house will be worth more, maybe we should look at a bigger place lol... ontop of another crazy pc project comming up lol.
